idioms/ctor.md

```diff
@@ -3,34 +3,89 @@
 ## Description
 
 Rust does not have constructors as a language construct. Instead, the
-convention is to use a static `new` method to create an object.
+convention is to use an [associated function][] `new` to create an object:
 
-## Example
+```rust
+/// Time in seconds.
+///
+/// # Example
+///
+/// ```
+/// let s = Second::new(42);
+/// assert_eq!(42, s.value());
```

Comment:
the `value()` function doesn't exist.
It is also present in other 2 examples. So I think that rustdoc examples are not tested.

Reply:
Good catch! 

Conflict: (a) use `pub value` to keep the code simple but (unnecessarily) promote using `pub` fields, or (b) add `fn value` and make the code more complicated. Opinions?

Reply:
I think "b" makes more sense.
If you use "a" you can create Second with invalid values by bypassing the new method